### PR TITLE
types: hardware and system additions for bios and site related changes

### PR DESCRIPTION
**Pull request by kwaegema**
Also fixes type of `bmc` console which was mistakenly set to `structure_dpc_console`.

### CODE REVIEW

File: quattor/types/site.pan [hunk 1]
```diff
@@ -0,0 +1,9 @@
+declaration template quattor/types/site;
+
+@documentation{
+    site specific system structure entries
+}
+type structure_system_site = {
```
Comment (jrha) [inline quattor/types/site.pan]: Any example of what might go in this empty type?

Comment (kwaegema) [reply quattor/types/site.pan]:
```
declaration template quattor/types/site;

include 'licenses/schema';

@documentation{
    UGent specific: Basic information needed for vsc-manage interaction.
}
type rundeck_info = { 
    @{Who governs an opennebula cluster being one, ...}
    "onemaster" ? string
};

type structure_system_ugent = { 
    "rundeck" ? rundeck_info
};

type structure_system_site = { 
    "quattorid"     : string = "-1"
    "licenses"      ? structure_license[]
    "ugent"         ? structure_system_ugent
};
```

Comment (jrha) [reply quattor/types/site.pan]: Ok thanks, `licenses` might be worth moving to an upstream schema though!

**Comment by kwaegema** [comment]
@jrha added the license structure

File: quattor/types/license.pan [hunk 2]
```diff
@@ -0,0 +1,9 @@
+declaration template quattor/types/license;
+
+type structure_license = {
```
Comment (ned21) [inline quattor/types/license.pan]: s/license/licence/ 

license is the verb not the noun so I think this needs to change throughout the file. :(

Comment (kwaegema) [reply quattor/types/license.pan]: Ah, british english :smile: will run the search-and-replace ;-)

**Pull request merged**